src/media/js/views/search.js

```diff
@@ -117,6 +117,7 @@ define('views/search',
         };
 
         var query = $('#search-q').val() || $('#search-q-desktop').val();
+        var query = query.replace( /  +/g, ' ' );
```

Comment:
@spasovski I was about to reply that waseem18 is collapsing sequential whitespace when I realized that \s\s+ is actually not sufficient. + means "at least 1" -- which means that (for example) \t would *not* get replaced by ' ' -- which means it would not be caught in the if statement on line 130 and would pass through as a valid search string. @waseem18 I think this regex works for consecutive whitespace strings with length >= 2, but it doesn't work for single non ' ' whitespace (like \t) strings. I don't know if it's worth using a regex to force all \s to '  ' -- maybe it is -- in which case you could use /\s\s*/ /g.